content/es/registrar/pr.md

```diff
@@ -3,15 +3,22 @@
 state_name = "Puerto Rico"
 state_abbreviation = "pr"
 is_state = "false"
-registration_type = "in-person"
+registration_type = "online"
 translationKey = "Puerto Rico"
 type = "register"
 hp_link = "https://ww2.ceepur.org/Home/Index?aspxerrorpath=/es-pr/Paginas/default.aspx"
+registration_link = "https://ere.ceepur.org/"
 more_info_link = "https://ww2.ceepur.org/es-pr/Paginas/cee-english.aspx"
 confirm_registration_link = "https://consulta.ceepur.org/"
+national_voter_registration_form_accepted = "false"
 hp_link_translated = "true"
 confirm_registration_link_translated = "true"
+registration_link_translated = "true"
+more_info_link_translated = "true"
 default_ip_deadline = "hasta 50 días antes del día de las elecciones"
+default_online_deadline = "hasta 50 días antes del día de las elecciones"
+registration_link_override = "https://ere.ceepur.org/"
+confirm_registration_link_override = "https://consulta.ceepur.org/"
 more_info_link_override = "https://ww2.ceepur.org/es-pr/Paginas/default.aspx"
```

Comment:
just confirming... this link is different than the more info link in the state.json file so this is ok to stay since it is a different link the the json file

Reply:
Yes, correct.